init-ota.h

```diff
@@ -0,0 +1,90 @@
+#ifndef __INIT_OTA__
+#define __INIT_OTA__
+
+const char* espUpdatesFile = "/updates.json";
+
+// OTA Logic 
+bool execOTA(JsonObject& json) 

```

Comment:
can we have more verbose nape for json argument
I suppose that **updateInfo** if good enough